swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition = 0):
+        super().__init__(category, condition)
+
+    def __str__(self):
+        return "The finest clothing you could wear."
```

Comment:
Having category set with a default parameter here would allow users to change the category of a Clothing instance like so `shirt = Category("Electronics", 5)`.

To prevent the user from changing the category, we can make `category` a default argument in the super constructor instead. This will ensure that every instance of a Clothing item will have a category of Clothing.

```suggestion
class Clothing(Item):
    def __init__(self, condition = 0):
        super().__init__(category = "Clothing", condition)

    def __str__(self):
        return "The finest clothing you could wear."
```